src/Files.Uwp/MultilingualResources/Files.vi.xlf

```diff
@@ -276,23 +276,23 @@
         </trans-unit>
         <trans-unit id="BaseLayoutItemContextFlyoutPinToFavorites.Text" translate="yes" xml:space="preserve">
           <source>Pin to Favorites</source>
-          <target state="translated">Gắn vào Ưa thích</target>
+          <target state="translated">Ghim vào Ưa thích</target>
         </trans-unit>
         <trans-unit id="BaseLayoutItemContextFlyoutProperties.Text" translate="yes" xml:space="preserve">
           <source>Properties</source>
           <target state="needs-review-translation" state-qualifier="tm-suggestion">Thuộc tính</target>
         </trans-unit>
         <trans-unit id="WelcomeDialog.Title" translate="yes" xml:space="preserve">
           <source>Welcome to Files!</source>
-          <target state="needs-review-translation">Chào mừng đến Files!</target>
+          <target state="needs-review-translation">Chào mừng bạn đến với Files!</target>
```

Comment:
You have some lines, like this, with the state "needs-review-translation". This state is added automatically when originally modifying the original string, or added manually by the translator when he/she is not sure about the result of the translation. Is the second scenario your case? If not, please replace in this line and all others "needs-review-translation" where is not the case with the state "translated".

Reply:
I've forgot to mention you @YT-Advanced , sorry.